.github/workflows/podman-test.yaml

```diff
@@ -22,4 +22,4 @@ jobs:
 
     - name: Run Integration tests
       run: |
-        make test-integration-podman
+        GINKGO_TEST_ARGS='-p' make test-integration-podman
```

Comment:
Whey here and not directly in the Makefile's `test-integration-podman` entry? I would be happy to run in parallel locally by running `make test-integration-podman` only

Reply:
I was about to ask someone else to try it locally ;-)
I had some issues locally with some tests randomly failing (and not always the same test or reason): sometimes the timeout waiting for the Dev mode, sometimes it had network issues trying to access the NPM Registry, ...
Can you try running this same command locally to see if you have similar issues?

Reply:
Initially, I wanted to revert this and look into those parallelization issues at a later time (to unblock https://github.com/redhat-developer/odo/pull/6485, which was the initial goal of this PR on isolation). But I thought it would still be helpful to run them in parallel on GitHub, as it divided the time of this step by almost 2.

Reply:
I tried to run them locally (with 11 parallel tests), but many tests are failing, with the pod not being started correctly. I have to check the reason of the failure

Reply:
FYI, I updated the test to run with 10 parallel nodes on GitHub, and now the whole suite is running in ~10 minutes. See https://github.com/redhat-developer/odo/actions/runs/3969461632/jobs/6830700899
It started failing on GitHub too with more parallel nodes, but for other reasons (e.g.: https://github.com/redhat-developer/odo/actions/runs/3969327423/jobs/6803669840#step:5:424)..
What do you think of looking into these parallelization issues in a separate issue, so I can unblock the tests of https://github.com/redhat-developer/odo/pull/6485?